general-remarks.md

```diff
@@ -121,6 +125,12 @@ Or
 | `6` | Validation error - e.g. Invalid value "XXX" of field "YYY". |
 | `7` | Processing error - e.g. Processing of the booking would violate some internal PMS limitation. |
 | `8` | Invalid authorization |
+| `9` | Rate error - e.g., rate plan XX prices updates cannot be accepted due to the settings in the channel manager extranet. |
+| `10` | Category error - e.g., space category XX availability updates cannot be accepted due to the settings in the channel manager. | 
```

Comment:
missing one word: space category XX availability updates cannot be accepted due to the settings in the channel manager extranet.